### PR TITLE
 Update changelog for v1.5.6 

### PR DESCRIPTION
**Pull request by meln1k**
updating changelog

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -1,3 +1,16 @@
+## Changes from 1.5.5 to 1.5.6
+Bugfix release
+
+### Fixed issues
+
+- [MARATHON-7990](https://jira.mesosphere.com/browse/MARATHON-7990) Ease up on the toString for Group and log clashing ids. (#5857)
+- [COPS-2072](https://jira.mesosphere.com/browse/COPS-2072) Expose `containerPort` as `$PORT_NAME` env vars. (#5892)
```
Comment (zen-dog) [inline changelog.md]: s/env vars/in environment variables

File: changelog.md [hunk 2]
```diff
@@ -1,3 +1,16 @@
...
+- [MARATHON-7914](https://jira.mesosphere.com/browse/MARATHON-7914) Migration from joda-time to java.time. (#5886)
+- [MARATHON-8005](https://jira.mesosphere.com/browse/MARATHON-8005) Backport of OffsetDateTime json serialization is different form Timestamp serialization. (#5917)
```
Comment (zen-dog) [inline changelog.md]: s/Backport of/Fixing OffsetDateTime json serialization that was different...

File: changelog.md [hunk 3]
```diff
@@ -1,3 +1,16 @@
+## Changes from 1.5.5 to 1.5.6
+Bugfix release
+
+### Fixed issues
+
+- [MARATHON-7990](https://jira.mesosphere.com/browse/MARATHON-7990) Ease up on the toString for Group and log clashing ids. (#5857)
```
Comment (zen-dog) [inline changelog.md]: s/toString/`toString` method

**Comment by alenkacz** [comment]
this is not supposed to go to master? Only 1.5?

**Pull request merged**